cmake/ConfigurationDependencies.cmake

```diff
@@ -70,16 +70,15 @@ o2_define_bucket(
     DEPENDENCIES
     ${CURL_LIBRARIES}
     ${Boost_PROGRAM_OPTIONS_LIBRARY}
-    ${Common_LIBRARIES}
     ${MYSQL_LIBRARIES}
 
     SYSTEMINCLUDE_DIRECTORIES
     ${Boost_INCLUDE_DIR}
     ${CURL_INCLUDE_DIRS}
-    ${Common_INCLUDE_DIRS}
```

Comment:
@ktf This will break `src/Backends/MySql/MySqlBackend.cxx` which requires `Common/GuardFunction.h`

Reply:
Actually the `GuardFunction` is not used so this include can be delete

Reply:
Looks like that's a leftover import. We can remove it. It's just strange that the CI build isn't broken

Reply:
Shouldn't we use [ScopeExit](http://www.boost.org/doc/libs/1_66_0/libs/scope_exit/doc/html/index.html) which is roughly the same thing from boost?

That said, where is `GuardFunction` actually used? I think this is a spurious include. Can you check if my new commit works (I do not have mysql installed on my laptop)?

This is also another great example of why the backend should be split from the actual frontend API.

Reply:
... and of course we should disable / drop the mysql backend now that we decided to go for consul.

Reply:
@ktf indeed, the code for mysql can be disabled but I would do it in another PR.

Reply:
@Barthelemy agree. I will wait for @awegrzyn and @PascalBoeschoten to give green light to this and then I will let them to disable the MySQL part.

Reply:
@ktf We'll get rid of more than just MySQL ;)